src/components/JustNotSorry.js

```diff
@@ -41,14 +41,24 @@ class JustNotSorry extends Component {
     this.setState({ parentNode: {}, warnings: [] });
   }
 
-  updateWarnings(elem, patterns) {
-    const newWarnings = findRanges(elem, patterns);
-    this.setState({ parentNode: elem.parentNode, warnings: newWarnings });
+  updateWarnings(email, patterns) {
+    const newWarnings =
+      email.children.size > 0
+        ? Array.from(email.children)
+            .filter((node) => node.text !== '')
```

Comment:
`node.text` seems to always return undefined.  Should this be `node.textContent` instead?

Reply:
Nice catch again... My html web api is very rusty
Thank you for catching it!